CHANGES.rst

```diff
@@ -20,6 +20,7 @@ Internationalization
 Improvements
 ^^^^^^^^^^^^
 
+- Grant exceptions and allow individuals to modify registration (:issue:`5264`, thanks :user:`Thanhphan1147`)
```

Comment:
This needs to go at the bottom of the improvements section.

Reply:
Thanks, I moved it to the bottom!